index.bs

```diff
@@ -681,8 +701,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
         [=hash of the serialized client data=]
     : {{tokenBinding}}
     :: The [=Token Binding ID=] associated with |callerOrigin|, if one is available.
-    : {{CollectedClientData/extensions}}
+    : {{CollectedClientData/clientExtensions}}
     :: |clientExtensions|
```

Comment:
It's now as follows, per your suggestion:
typedef record<DOMString, any> AuthenticationExtensions;